### PR TITLE
[Regression](planner) Add tpch plan check

### PR DESCRIPTION
**Pull request by Kikyou1997**
# Proposed changes

Issue Number: noissue

## Problem Summary:

Add plan check for tpch so that we can sense any changes to the final plan opportunely.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
@morrySnow

**Comment by morrySnow** [comment]
please fix license

**Comment by Kikyou1997** [comment]
@morrySnow  review please

File: regression-test/suites/tpch_sf1/script/README [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Manual
+
```
Comment (morrySnow) [inline regression-test/suites/tpch_sf1/script/README:2]: We need a more detailed README. And make this script a little more general for any other case

**Comment by Kikyou1997** [comment]
@dataroaring @sodamnsure help to review pls

**Review by kpfly — DISMISSED** [review]
LGTM

File: regression-test/suites/tpch_sf1/load.groovy [hunk 2]
```diff
@@ -77,4 +77,5 @@ suite("load") {
     def table = "revenue1"
     sql new File("""${context.file.parent}/ddl/${table}_delete.sql""").text
     sql new File("""${context.file.parent}/ddl/${table}.sql""").text
+    sleep(60000)
```
Comment (morrySnow) [inline regression-test/suites/tpch_sf1/load.groovy:80]: we need a comment on it

**Review by morrySnow — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by kpfly — APPROVED** [review]
LGTM

**Pull request merged**